Database-Features/removing-corrupt-data.md

```diff
@@ -45,7 +45,7 @@ set verbose on;
 @/tmp/check_all_columns.sql;
 ```
 
-To execute the file, run `exaplus -u <username> -p <passwd> -c <connection string> -f corrupt_data_check.sql`. 
+To execute the file, run `exaplus -u <username> -p <passwd> -c <connection string> -f corrupt_data_check.sql`. Database user requires SELECT ANY DICTIONARY system privilege for executing this script, see [Details on Rights Management](https://docs.exasol.com/db/latest/database_concepts/privileges/details_rights_management.htm).
```

Comment:
@allipatev the user also needs the permission to SELECT every table in the database. I would rather state that the user must be a DBA for simplification.

Reply:
Agree, it seemed obvious to me yesterday ...